datamade/la-metro-councilmatic
# Updating report actions table with more information

## Overview

- [x] Changes "Committee" to "Meeting"
- [x] Links to relevant event instead of committee overview page
- [x] Displays "Cancelled" a la the events page

### Checklist


### Demo

<img width="857" alt="Screen Shot 2019-05-31 at 2 54 51 PM" src="https://user-images.githubusercontent.com/6961258/58731374-2505cd80-83b4-11e9-9cfd-0f1ae35619e2.png">
<img width="859" alt="Screen Shot 2019-05-31 at 2 54 59 PM" src="https://user-images.githubusercontent.com/6961258/58731375-2505cd80-83b4-11e9-85b6-bebaadd2bc2f.png">

### Notes

I used template tags instead of a manager because I wanted the function to occur per-object, not on a QuerySet. This seemed like the least complex way to do that.

## Testing Instructions

- Navigate to a piece of legislation
- Hard refresh the page
- Confirm link goes to appropriate event agenda
- Confirm event status is correct
- Repeat with a few different pages

Handles #348
